package.json

```diff
@@ -16,7 +16,6 @@
     "clone-deep": "^4.0.1",
     "node-sass": "4.12.0",
     "npm-run-all": "^4.1.5",
-    "plotly.js-basic-dist": "^1.47.3",
```

Comment:
deleted duplicate (plotly.js-basic-dist import is also listed on line 32/33)

Reply:
sounds good